src/Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.csproj

```diff
@@ -27,11 +27,12 @@
     <PackageReference Include="XliffTasks" Version="$(XliffTasksPackageVersion)" PrivateAssets="All" />
 </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.5' ">
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETStandard' ">
+    <PackageReference Include="NETStandard.Library" Version="2.0.0" />
     <PackageReference Include="System.Diagnostics.Process" Version="4.1.0" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net46' ">
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">
```

Comment:
I'm not crazy about this pattern because it works for item groups but not property groups.

Reply:
But is it worth it if you don't have to update every condition when you move from `netstandard2.0` to `netstandard2.1` or from `net46` to `net461`?  That's the reason I prefer it...

Reply:
The subtlety of msbuild evaluation order is something I'd rather not think about when reading these conditions. If there's a mismatch, the bug is obvious. I worry that cargo cult will consider this "better", then scream "bug" when it fails on a property group. We should document and use the pattern that works for properties and items.